### DISCUSSION TITLE
The pagespeed insights module displays API error 400

### DISCUSSION THREAD

**Discussion by RavelloH** [lowlighter/metrics#1159] [Q&A] [answered]
Recently, the phenomenon of API error 400 began to appear in my pagespeed insights, as shown in the following screenshot:

![Screenshot_20220726_182255](https://user-images.githubusercontent.com/68409330/180984245-e97b2fe4-717b-478b-9c93-f30a95a2d679.jpg)

I also checked several other accounts, and there were such problems, which worked normally before.....

My settings are as follows:
```
https://metrics.lecoq.io/ravelloh?template=classic&base.header=0&languages=1&introduction=1&gists=1&followup=1&lines=1&projects=1&discussions=1&notable=1&pagespeed=1&languages.limit=8&languages.sections=most-used&languages.colors=github&languages.threshold=0%25&languages.indepth=false&languages.categories=markup%2C%20programming&languages.recent.categories=markup%2C%20programming&languages.recent.load=300&languages.recent.days=14&introduction.title=true&followup.sections=repositories&projects.limit=4&projects.descriptions=true&notable.repositories=false&pagespeed.url=.user.website&pagespeed.detailed=true&pagespeed.screenshot=true&config.timezone=Asia%2FShanghai&config.display=large
```

How to solve this problem? thank you!

**Comment by lowlighter** [comment, marked as answer]
Sorry last time I mistakenly revoked the pagespeed token, so it was erroring with the following error:
```
Jul 27 15:08:03 metrics docker[475750]:           code: 400,
Jul 27 15:08:03 metrics docker[475750]:           message: 'API key expired. Please renew the API key.',
```

It should be fixed now, thanks for reporting it 👍

**Reply by RavelloH** [reply]
Thank you for your response, this project is really great!

**Reply by lowlighter** [reply]
Thanks ❤️ !
